contrib/coreos/user-data.example

```diff
@@ -20,6 +10,46 @@ coreos:
   units:
   - name: etcd.service
     command: start
+    content: |
```

Comment:
If only there was a way we could deploy this with `deisctl`... We'll never be able to tune this in future releases, which could be a concern.